packages/server/src/routes/me/mint.ts

```diff
@@ -88,6 +84,27 @@ async function _mintReferralBonus(
   };
 }
 
+async function _mintInvitedBonus(
+  loggedInMember: firestore.DocumentSnapshot,
+  amount: Big
+): Promise<MintReferralBonusPayload> {
+  if (!loggedInMember.get("invite_confirmed")) {
+    throw new NotVerifiedError(loggedInMember);
+  }
+  const createdAt = loggedInMember.createTime;
+  if (!createdAt) {
+    // TODO create some type for generic unspecified error tht we do not expect client to interpret,
+    // because we don't expect it to happen and want devs to be alerted if it does.
+    throw Error("Logged in member create time did not exist, there is a bug.");
+  }
+  const allowedAmount = Config.getInvitedBonus(createdAt.toMillis());
+  throwIfTooLarge(amount, allowedAmount);
+  return {
```

Comment:
Can we add the already minted check here too? Just in case client sends duplicate